### PR TITLE
feat: legend label theme

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

```ts
type FontTheme = {
  fontSize?: number;
  fontFamily?: string;
  fontWeight?: string | number;
  color?: string;
};

type LegendTheme = {
  label?: FontTheme;
};
```

image

image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
[11/10] 리뷰완료
고생하셨습니다!

File: src/component/legend.ts [hunk 1]
```diff
@@ -2,23 +2,27 @@ import Component from './component';
...
 import { LegendData, LegendModel } from '@t/components/legend';
 import {
+  getLegendItemHeight,
   LEGEND_CHECKBOX_SIZE,
   LEGEND_ICON_SIZE,
   LEGEND_ITEM_MARGIN_X,
-  LEGEND_LABEL_FONT,
   LEGEND_MARGIN_X,
-  LEGEND_ITEM_HEIGHT,
+  // LEGEND_ITEM_HEIGHT,
```
Comment (jungeun-cho) [inline src/component/legend.ts:10]: 사용하지 않으면 지워도 될것같아요!

**Review by jungeun-cho — COMMENTED** [review]
[11/10] 이견없습니다. 수고하셨어요~!

**Review by jungeun-cho — APPROVED** [review]

**Comment by shiren** [comment]
폰트 바꾸니 늬낌이 완전 다르네.. LGTM

**Pull request merged**